### DISCUSSION TITLE
Need Accessibility support.

### DISCUSSION THREAD

**Discussion by upondeep** [NG-ZORRO/ng-zorro-antd#6658] [General]
Need to support ARIA label and tab focus, Material and Bootstrap all support them, Zorro should too if want to be competitive.

**Comment by alehar9320** [comment]
It's unfortunate that the library is missing accessibility support, as it is nowadays a requirement for successful enterprise usage (risk of lawsuits / losing out on deals). Is anyone aware of if there are any updates as to when improvements may be made?

The last record I can find is https://github.com/NG-ZORRO/ng-zorro-antd/issues/3667, but it's from 2019. There are also a bunch of stale pull requests.
